lms/static/sass/_build-lms-v2.scss

```diff
@@ -3,7 +3,22 @@
 // Version 2 - introduces the Pattern Library
 
 
-// Configuration
```

Comment:
Hey @talbs! How's your awesome fort coming along? More importantly, did you see the Rogue One teaser yet? It had me at AT-ATs!